packages/dd-trace/src/span_processor.js

```diff
@@ -42,6 +44,7 @@ class SpanProcessor {
 
       for (const span of started) {
         if (span._duration !== undefined) {
+          this._llmobs.process(span)
```

Comment:
Could this be decoupled with a channel?

Reply:
yes, i think it can! for clarification, this will mean i'll initialize the LLMObs span processor in the SDK (next PR) and add `processor.format` as a subscriber to that channel when the SDK is initialized. It should actually give me better/direct access to the LLMObs span processor's writer